products/appengine/terraform.yaml

```
@@ -109,7 +109,30 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         primary_resource_id: "web_service"
         vars:
           bucket_name: "appengine-test-bucket"
-
+  ServiceSplitTraffic: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: "{{service_id}}"
+    import_format: ["apps/{{project}}/services/{{service_id}}"]
+    mutex: "apps/{{project}}"
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      custom_delete: templates/terraform/custom_delete/skip_delete.go.erb
+      test_check_destroy: templates/terraform/custom_check_destroy/skip_delete_during_test.go.erb
+    properties:
+      id: !ruby/object:Overrides::Terraform::PropertyOverride
+        name: 'service_id'
+        ignore_read: true  
+      split: !ruby/object:Overrides::Terraform::PropertyOverride
+        ignore_read: true    
+      split.allocations: !ruby/object:Overrides::Terraform::PropertyOverride
+        custom_flatten: templates/terraform/custom_flatten/float64_to_string.go.erb
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "app_engine_service_split_traffic"
+        vars:
```

```suggestion
        primary_resource_id: 'myapp'
        vars:
```

This should resolve the issue you were seeing. It was caused when the test framework effectively ran `terraform import google_app_engine_service_split_traffic. {{id}}` instead of `terraform import google_app_engine_service_split_traffic.myapp {{id}}`

---

Cool, thanks I have committed the suggestion.